manager/src/test/java/com/redhat/service/bridge/manager/BridgesServiceTest.java

```diff
@@ -193,4 +198,27 @@ public void getBridge() {
     public void getBridge_bridgeDoesNotExist() {
         assertThatExceptionOfType(ItemNotFoundException.class).isThrownBy(() -> bridgesService.getBridge("foo"));
     }
+
+    @Test
+    public void testDeleteBridge() {
```

Comment:
You could add a test for the the `Bridge` is `FAILED` too.

Reply:
That is when it is `PROVISIONING` not `FAILED`. When `FAILED` you should be allowed to delete it, but I don't see a test for that... however please point me to one. I could not see it.

Reply:
okies, I miss took your earlier comment. I thought you are asking to add `failure` usecase. I will add it.

Reply:
I had added new testcase for `FAILED` scenerio.
https://github.com/vaibhavjainwiz/sandbox/blob/f68de90bd03dc5bf56f0059025d09312f694e6b6/manager/src/test/java/com/redhat/service/bridge/manager/BridgesServiceTest.java#L213